### PR TITLE
add models

### PR DESCRIPTION
**Pull request by Pavel-An**
Добавил модели 
Project
ProjectUser

Связал модели User и  Project через ProjectMember

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Project < ApplicationRecord
+  has_many :project_members
```
[the remark this answers is not in the document]
Comment (fooooxmr) [reply app/models/project.rb:6]: According to Rails Style Guide - https://courses.bigbinaryacademy.com/learn-rubyonrails/order-of-items-on-active-record-models/

Associations should be listed before the validations.

File: .irb_history [hunk 2]
```diff
@@ -0,0 +1,56 @@
+User.all
+u = User.first
```
[the remark this answers is not in the document]
Comment (fooooxmr) [reply .irb_history:2]: This file should be removed from git. Delete this file locally and push changes withour it.

[the remark this answers is not in the document]
Comment (Pavel-An) [reply .irb_history]: Done

File: app/models/project.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pavel-An) [reply app/models/project.rb:8]: Done

File: app/models/user.rb [hunk 3]
```diff
@@ -3,4 +3,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
```
[the remark this answers is not in the document]
Comment (Pavel-An) [reply app/models/user.rb:6]: Done

**Review by fooooxmr — APPROVED** [review]

**Pull request merged**